sdk/python/feast/repo_config.py

```diff
@@ -63,7 +65,7 @@ class RepoConfig(FeastBaseModel):
     provider: StrictStr
```

Comment:
I think we'll need defaults for provider and project in order for `RepoConfig()` with no arguments to work